### DISCUSSION TITLE
Question about decimal positions.

### DISCUSSION THREAD

**Discussion by maurodeoliveira** [d3/d3#3749] [Q&A] [answered]
Hello, I am using treemap and i have one question about it. In all rects that treemap show i can see value with 2 decimal positions. If i want to show the same value but without decimal positions. Can i do this? Thanks a lot.

**Comment by Fil** [comment, marked as answer]
Can you share some code or be more specific about which example you're starting from? If the value is displayed with [d3.format](https://d3js.org/d3-format), try to use a format such as "d" which shows integers.
